op-chain-ops/genesis/config.go

```diff
@@ -470,6 +470,7 @@ func NewDeployConfigWithNetwork(network, path string) (*DeployConfig, error) {
 // L1Deployments represents a set of L1 contracts that are deployed.
 type L1Deployments struct {
 	AddressManager                    common.Address `json:"AddressManager"`
```

Comment:
This should also have mips and alphabet?

Reply:
There's no reason to refer to them directly - they should just be accessed via the dispute game factory and the games it creates.

Reply:
Even though they can be considered "L1 Deployments", we should follow our practice of only including items that are used for testability and maintainability. As such, we shouldn't need to include MIPS and Alphabet themselves